linters/cpp/exclusions.py

```diff
@@ -129,25 +129,65 @@
 	'tests/catapult/utils/CountOfTests.cpp': 'catapult/types.h',
 	'tests/catapult/utils/MacroBasedEnumTests.cpp': 'catapult/utils/MacroBasedEnumIncludes.h',
 	'tests/catapult/utils/TraitsTests.cpp': 'catapult/utils/traits/Traits.h',
-	'tests/catapult/utils/StlTraitsTests.cpp': 'catapult/utils/traits/StlTraits.h'
+	'tests/catapult/utils/StlTraitsTests.cpp': 'catapult/utils/traits/StlTraits.h',
+
+	'tests/int/node/basic/LocalNodeBasicTests.cpp': 'tests/int/node/test/LocalNodeBasicTests.h',
+	'tests/int/stress/NemesisBlockLoaderIntegrityTests.cpp': 'catapult/extensions/NemesisBlockLoader.h',
+	'tests/int/stress/UtUpdaterIntegrityTests.cpp': 'catapult/chain/UtUpdater.h'
 }
 
 PLUGINS_FIRSTINCLUDES = {
```

Comment:
if we are using clang_format, should we mostly remove these and let it order?

Reply:
ofc, would need to rip out all headers ordering/checking, I updated the rules, just to make sure what clang did with _our rules_ makes some sense